libs/langchain/langchain/llms/petals.py

```diff
@@ -142,6 +142,7 @@ def _call(
         """Call the Petals API."""
         params = self._default_params
         params = {**params, **kwargs}
+        # self.huggingface_api_key = cast(SecretStr, self.huggingface_api_key)
```

Comment:
delete

Reply:
Sorry, my bad! This happened  because I was referencing this merged [PR](https://github.com/gautamanirudh/langchain/blob/6a997d17a082c24c851db8b20e0624e8d5d99d44/libs/langchain/langchain/llms/ai21.py#L146) and the author was facing `mypy` error without using `cast`. However, when I tested my implementation , I didn't encounter the error but forgot to remove the `cast`.